xc/device.ml

```diff
@@ -1912,6 +1912,30 @@ let stop ~xs ~qemu_domid domid  =
     stop_vgpu ();
     stop_qemu ()
 
+let is_hvm_linux domid =
```

Comment:
None means that true or false couldn't be determined, but functionally it behaves like false, so I can simplify this further by making is_hvm_linux return a boolean as you suggest.